polib.py

```diff
@@ -31,28 +31,14 @@
 # the default encoding to use when encoding cannot be detected
 default_encoding = 'utf-8'
 
-# python 2/3 compatibility helpers {{{
+# python 2/3 compatibility helpers (obsolescent) {{{
+text_type = str
```

Comment:
```suggestion
```

`text_type` alias shouldn't be needed anymore without Python 2

Reply:
Users may still be using them, so I don't believe they can be removed for now.